k8s/scripts/inspect.sh

```diff
@@ -54,7 +54,7 @@ function collect_args {
 
 function collect_cluster_info {
   log_info "Copy k8s cluster-info dump to the final report tarball"
-  k8s kubectl cluster-info dump --output-directory "$INSPECT_DUMP/cluster-info" &>/dev/null
+  k8s kubectl cluster-info dump --all-namespaces --output-directory "$INSPECT_DUMP/cluster-info" &>/dev/null
```

Comment:
Is this the part where we switch to collect logs from all namespaces? I'm not too sure about this script is used by actual users as well. We shouldn't get logs to their workloads which might contain secrets, sensitive info etc.

Reply:
Good point, I assumed that it's only used by the CI. We could add an option called ``--verbose`` or ``--detailed``. It can be a follow-up to avoid delaying this PR.

Reply:
let's make this a configurable flag of the script. 
For the inspection report, we can then enable this flag (or specify all of our namespace).
I'd tackle this in a separate PR.